### PR TITLE
 Fix parsing of Depfu PR comments to reflect added and ignore removed gems

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

File: github-labeling-bot/.gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 bot/config/config.yml
+.bundle
```
Comment (Ana06) [inline github-labeling-bot/.gitignore]: how is this related? :confused:

Comment (ChrisBr) [reply github-labeling-bot/.gitignore]: It is not but do you really expect me to create a dedicated PR for this? It is in a dedicated commit...

**Comment by ChrisBr** [comment]
> [nitpick] git-cop wouldn't like your commit

Because it is too long or what is your problem with it? Anyway we decided to not use git cop anymore ...
